pkg/deploy/deploy.go

```diff
@@ -160,52 +114,6 @@ func upsertNonUniqueNameConfig(client dtclient.ConfigClient, apiToDeploy api.API
 	return client.UpsertConfigByNonUniqueNameAndId(apiToDeploy, entityUuid, configName, []byte(renderedConfig))
 }
 
-func deploySetting(settingsClient dtclient.SettingsClient, properties parameter.Properties, renderedConfig string, c *config.Config) (*parameter.ResolvedEntity, []error) {
-	t, ok := c.Type.(config.SettingsType)
-	if !ok {
-		return &parameter.ResolvedEntity{}, []error{fmt.Errorf("config was not of expected type %q, but %q", config.SettingsTypeId, c.Type.ID())}
-	}
-
-	scope, err := extractScope(properties)
-	if err != nil {
-		return &parameter.ResolvedEntity{}, []error{err}
-	}
-
-	entity, err := settingsClient.UpsertSettings(dtclient.SettingsObject{
-		Id:             c.Coordinate.ConfigId,
-		SchemaId:       t.SchemaId,
-		SchemaVersion:  t.SchemaVersion,
-		Scope:          scope,
-		Content:        []byte(renderedConfig),
-		OriginObjectId: c.OriginObjectId,
-	})
-	if err != nil {
-		return &parameter.ResolvedEntity{}, []error{newConfigDeployErr(c, err.Error())}
-	}
-
-	name := fmt.Sprintf("[UNKNOWN NAME]%s", entity.Id)
-	if configName, err := extractConfigName(c, properties); err == nil {
-		name = configName
-	} else {
-		log.Warn("failed to extract name for Settings 2.0 object %q - ID will be used", entity.Id)
-	}
-
-	properties[config.IdParameter], err = getEntityID(c, entity)
-	if err != nil {
-		return &parameter.ResolvedEntity{}, []error{newConfigDeployErr(c, err.Error())}
-	}
-
-	properties[config.NameParameter] = name
-
-	return &parameter.ResolvedEntity{
-		EntityName: name,
-		Coordinate: c.Coordinate,
-		Properties: properties,
-		Skip:       false,
-	}, nil
-
-}
-
 func getEntityID(c *config.Config, e dtclient.DynatraceEntity) (string, error) {
```

Comment:
this method and `extractScope` are Settings specific -> they should move to deploy/settings.go as well